MetaMask/metamask-extension
# [FLASK] Snaps Insight

## Explanation

All the logic and UI needed for the new Snaps Insight API.

## More Information

* Fixes [#655](https://github.com/MetaMask/snaps-skunkworks/issues/655)

## Manual Testing Steps

- Clone https://github.com/GuillaumeRx/snap-tx-insight
- `yarn install && yarn start` in the `snap-tx-insight` repo
- connect the snap (We have a bug right now, after clicking connect and allowing the snap the connect button will still appear, no worries
- Do a contract interaction (like a unsiwap swap)
- Look at the tx insight tab in the tx confirmation screen
